### ISSUE TITLE
Unhandled stream error.

### ISSUE THREAD

**Issue by replete** [mpezzi/gulp-a11y#1]
I'm trying to find an accessibility plugin that actually works. I have gulp experience, so it's not anything obvious.

```
[18:00:50] Starting 'a11y'...
stream.js:94
      throw er; // Unhandled stream error in pipe.
            ^
a11y: Command failed: /Users/X/Dropbox/Projects/XXX/Frontend/node_modules/gulp-a11y/node_modules/a11y/node_modules/phantomjs/lib/phantom/bin/phantomjs /Users/X/Dropbox/Projects/XXX/Frontend/node_modules/gulp-a11y/node_modules/a11y/audits.js {"delay":1,"url":"file:///Users/X/Dropbox/Projects/XXX/Frontend/dist/layout.html","width":1024,"height":768} --ignore-ssl-errors=true --ssl-protocol=tlsv1 --local-to-remote-url-access=true
Audit failed
```

Node v0.12.3

**Comment by ghost** [comment]
same here

**Comment by replete** [comment]
Project seems dead, so closing comment. Shame.
